src/room/usePTT.js

```diff
@@ -47,29 +49,35 @@ export function usePTT(client, groupCall, userMediaFeeds) {
     };
   }, [userMediaFeeds]);
 
-  const startTalking = useCallback(() => {
+  const startTalking = useCallback(async () => {
+    setState((prevState) => ({ ...prevState, pttButtonHeld: true }));
+    setUnmuteError(null);
```

Comment:
I might store the `unmuteError` in `state` instead of it's own `useState`.

Reply:
OK - I flipped back & forth between staying consistent and splitting it out since merging with prevState each time seemed quite boilerplate-y, but since it's already setting other state in one case anyway maybe consistency is better.